Changelog.md

```diff
@@ -4,24 +4,44 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - ReleaseDate
+## [0.34.0] - ReleaseDate
 
 ### Added
 
+- Update Vulkan-Headers to 1.2.203 (#477, #497, #504, #509, #514)
+- Add missing documentation to bitflag extension variants (#501)
 - Added `VK_KHR_present_wait` device extension (#493)
+- Added `VK_KHR_maintenance4` device extension (#489, #498)
+- Link `_len()` functions to their array-getter using intradoc-links (#490)
+- Added `VK_KHR_dynamic_rendering` device extension (#488)
 
 ### Changed
 
+- examples: Use `slice::from_ref` to not loose lifetime on nested slices (#513)
+- Simplify triangle example's vertex input state (#512)
 - Device extension `khr::PipelineExecutableProperties` and `khr::TimelineSemaphore` now expose `fn device()` instead of `fn instance()` (#499)
 - Changed `khr::PipelineExecutableProperties::new()` and `khr::TimelineSemaphore::new()` to take `instance` and `device` as arguments (#499)
+- Fix broken vulkan-tutorial link in README.md (#492)
+- Make `enumerate_physical_device_groups` unsafe (#491)
+- Added `Packed24_8` helper-type for constructing AS Instance bitfields, used in `AccelerationStructureInstanceKHR`, `AccelerationStructureSRTMotionInstanceNV` and `AccelerationStructureMatrixMotionInstanceNV` (#476)s (#490)
+- Link Vulkan directly, statically, by default. Enable the `loaded` feature to get old `libloading` behaviour (#457)
 
 ### Removed
 
 - Removed `device()` function from `khr::Synchronization2` device extension (#494)
 - Removed `instance()` function from `ext::ExtendedDynamicState`, `khr::PushDescriptor`, `ext::ToolingInfo` and `khr::GetPhysicalDeviceProperties2` instance extensions (#494)
-- Removed `device` argument from `ext::DebugMarkers::debug_marker_set_object_name` function (#494)
+- Removed `device` argument from `ext::DebugMarkers::debug_marker_set_object_name` function, `khr::PipelineExecutableProperties` and `khr::TimelineSemaphore` functions (#494, #499)
 - Removed `From<vk::Result>` trait for `VkResult` (#495)
 - Removed `instance` argument from `ext::DebugUtils::submit_debug_utils_message` function (#499)
+- Removed misleading `all()`/`-`/`-=` function/ops from bitflags (#478)
+
+### Internal
+
+Changes inside Ash (ie. the `generator`) that should not affect our public API:
+
+- generator: Use `"Backwards-compatible"` comment to omit deprecated aliases (#502)
```

Comment:
Isn't the removal of deprecated aliases a public change?

Reply:
I recall forgetting to update that PR title now, as it was indeed decided during that PR to just drop deprecated enum variants while at it instead of coming up with suboptimal and useless names.

Moved it up to the `Removed` section, and reworded in a way that end-users know how to deal with the now-removed variants (though I do hope nobody was actively using and perhaps allowing these `#[deprecated]` constants, except of course the ones that only got their missing docs added in #501).

Reply:
Dropped this entire section and moved the `Self` change - which is much larger than the title suggests again :disappointed: - to `### Changed`.